react/game/src/app/components/Navbar/components/Goback/styles.scss

```diff
@@ -0,0 +1,15 @@
+.goback-button {
+  composes: navbar-menu-link-button from "../../styles.scss";
```

Comment:
Delete this. U must receive the style as a prop in the component, and apply the two components